integration-tests/src/main/java/io/druid/testing/utils/TestQueryHelper.java

```diff
@@ -118,4 +123,27 @@ private String getBrokerURL()
   {
     return String.format("%s/druid/v2?pretty", broker);
   }
+
+  @SuppressWarnings("unchecked")
+  public int countRows(String dataSource, String interval)
+  {
+    TimeseriesQuery query = Druids.newTimeseriesQueryBuilder()
+                                  .dataSource(dataSource)
+                                  .aggregators(
+                                      ImmutableList.<AggregatorFactory>of(
+                                          new LongSumAggregatorFactory("rows", "count")
+                                      )
+                                  ).granularity(Granularities.ALL)
```

Comment:
I think it's more according to the style to move `.granularity(Granularities.ALL)` to the next line.